conf/messages

```diff
@@ -349,3 +349,4 @@ notStarted = Not yet started
 completed = Completed
 checkNow = Check
 inProgress = In progress
+underMaintenance = Under Maintenance
```

Comment:
Duplicate of:

`common.underMaintenance = Under maintenance`

https://github.com/hmrc/income-tax-submission-frontend/blob/e0fae9dfc06aced834b7abba53c0a201ebfd144a/conf/messages#L21

Reply:
done